changelog/17172.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+secrets/terraform: upgrade to v0.6.0
```

Comment:
I don't think we've been including CL entires for the plugin version upgrades like this. The entries are mostly focused on notable bug fixes, improvements, features that a customer would want to know about coming in v0.6.0.

Reply:
If there are none, then feel free to omit the CL entry.

Reply:
This plugin has no changes, would it be better to remove the changelog file entirely? I can fix this on my kerberos update PR which does have changes.

Reply:
Answered before I asked 😆

Reply:
Yep, feel free to remove it 👍  Technically, nothing has changed in the plugin that a user should know about.